eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/service/UserService.java

```diff
@@ -36,7 +36,8 @@ public void deleteUser(String id) {
   }
 
   public User updateUser(String id, User user) {
-    user.setId(id);
-    return userRepository.save(user);
+    User u = userRepository.findById(id).get();
```

Comment:
Ez így nem biztonságos, mert a u lehet null. Hívd meg a `findOne(id)`-t

Reply:
Szia Robi. A findOne-t azt nálunk nem engedi a findbyid-t viszont igen. Ezt a hibát kapjuk findone-ra (Inferred type 'S' for type parameter 'S' is not within its bound; should extend 'hu.flowacademy.meetingorganizer.persistence.model.User')

Reply:
Oké, látom mi a gond. Akkor legyen a `.get()` helyett `.orElseThrow()`

Reply:
Változtatás megtörtént.